### ISSUE TITLE
WD advagg: Reading from the file system failed. This can sometimes happen durning a deployment.

### ISSUE THREAD

**Issue by bradallenfisher** [elmsln/elmsln#1485]
When running drush en title is error. 

fixed in dev of advagg, may be moot if we are dropping it. 
https://www.drupal.org/node/2631444

**Comment by btopro** [comment]
we will drop SOME aspects of it but not all so may be worth updating

**Comment by btopro** [comment]
we already run a version newer then this issue 7.x-2.18+73-dev

**Comment by btopro** [comment]
rolled latest release here which isn't much different from our dev bc of the issues I was tracking associated w/ browser based caching of future assets
